applications/DEMApplication/custom_elements/spheric_particle.cpp

```diff
@@ -989,14 +988,14 @@ void SphericParticle::ComputeBallToRigidFaceContactForce(SphericParticle::Partic
                 GeometryFunctions::CrossProduct(AngularVel, actual_arm_vector, tangential_vel);
 
                 Quaternion<double> DeltaOrientation = Quaternion<double>::Identity();
-                GeometryFunctions::OrientationFromRotationAngle(DeltaOrientation, delta_rotation);
+                GeometryFunctions::OrientationFromRotationAngle(DeltaOrientation, negative_delta_rotation);
```

Comment:
Everything looks good to me. I want to just add a minor comment, what do you think about changing the name DeltaOrientation to NegativeDeltaOrientation? I think it would clarify the code.